### PR TITLE
Add AwsSES adapter & Fix exist spec bug

### PR DESCRIPTION
**Pull request by keizo3**

### CODE REVIEW

File: spec/deliver_later_strategy_spec.cr [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe "Deliver later strategy" do
 end
 
 private def use_custom_strategy(strategy)
-  CustomizedBaseEmail.configure do
+  CustomizedBaseEmail.configure do |settings|
```
Comment (keizo3) [inline spec/deliver_later_strategy_spec.cr]: Fix the following bugs
```
in spec/deliver_later_strategy_spec.cr:35: undefined local variable or method 'settings'
```

**Comment by paulcsmith** [comment]
Hello Keizo! I think this is wonderful work and I'm excited to have Amazon SES available to people.

Could you make this as a separate repo that you control? I made the mistake when writing https://github.com/thoughtbot/bamboo of making too many adapters in the core repo and I couldn't handle all the issues and pull requests because I wasn't using those adapters myself.

Instead if you make your own I could add a link to your repo's README so people can figure add the adapter and use it. This also results in less code being downloaded required by carbon as it grows!

Does that sound good to you?

**Comment by keizo3** [comment]
Hllo Paul
Thank you for your suggestion.
I understand separate repo is good idea  and you are too busy.

I will close this  PR and make an issues.
I create repo and comment in to the issue.

**Comment by paulcsmith** [comment]
Wonderful! Thank you so much :D

**Pull request closed without merging**